Sources/EpoxyLayoutGroups/Types/LayoutGroupUpdateAnimation.swift

```diff
@@ -0,0 +1,73 @@
+// Created by lindsay_pond on 5/13/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import UIKit
+
+/// Provides parameter values, which can be passed to customize the animation on an Group's animated update (`_setItems(::)`).
+public struct LayoutGroupUpdateAnimation {
+  public init(id: AnyHashable, animate: @escaping GroupAnimationClosure) {
+    self.id = id
+    self.animate = animate
+
+  }
+  public var id: AnyHashable
+  public var animate: GroupAnimationClosure
+}
+
+extension LayoutGroupUpdateAnimation {
+
+  // Creates a spring style `LayoutGroupUpdateAnimation` instance with default or custom values
+  /// - Parameters:
+  ///   - duration: a static TimeInterval for animation duration, with a default value of `0.5`
+  ///   - delay: a static TimeInterval for animation duration, with a default value of `0.0`
+  ///   - dampingRatio: a static value for spring damping ratio, with a default value of `1.0`
+  ///   - initialSpringVelocity: a static value for initial spring velocity, wiht a default value of `0.0`
```

Comment:
nit: I'm not quite sure what static means here, can we omit?
```suggestion
  ///   - duration: a TimeInterval for animation duration, with a default value of `0.5`
  ///   - delay: a TimeInterval for animation duration, with a default value of `0.0`
  ///   - dampingRatio: a value for spring damping ratio, with a default value of `1.0`
  ///   - initialSpringVelocity: a value for initial spring velocity, wiht a default value of `0.0`
```